pyoverkiz/client.py

```diff
@@ -114,16 +114,16 @@ class OverkizClient:
     _refresh_token: str | None = None
     _expires_in: datetime.datetime | None = None
     _access_token: str | None = None
-    _ssl_context: ssl.SSLContext | None = None
+    _ssl: ssl.SSLContext | bool = True
```

Comment:
Should we include this as breaking change?

Reply:
It is a private setter so technically not a breaking change, but good to mention it. Here I am most afraid of breaking anything by the way, so will need to do proper testing before adding to HA.